src/pages/learn.js

```diff
@@ -7,19 +7,36 @@ import { HeartwoodDispatchContext } from "../state/HeartwoodContextProvider"
 
 import "../styles/video.css"
 
-export default () => {
+const Learn = ({location}) => {
+  const state = useContext(HeartwoodStateContext)
   const dispatch = useContext(HeartwoodDispatchContext)
 
+  // Jacob, make sure to turn this back to || false when you are done testing
+  const { showPage } = location.state || true
+
   return (
     <div className="flex flex-col w-full h-screen bg-base">
-      <Navbar />
-      <div className="flex flex-row items-center justify-center w-full h-full ">
-        <div className="flex flex-row w-full px-10">
-          <div className="flex flex-col w-1/4">
-            <div className="m-3 bg-black rounded-lg">
-              <div className="relative rounded-lg pb-3/4">
-                <div className="absolute object-cover w-full h-full">
-                  <Video className="rounded-lg" />
+      {showPage ?  (
```

Comment:
None of this. Do a `useEffect` or you're going to have an issue with window availability.